Content/.template.config/template.json

```diff
@@ -125,11 +138,15 @@
           "condition": "(Fulma != \"login\")"
         },
         {
-          "exclude": "**/Dockerfile",
+          "exclude": "Dockerfile",
           "condition": "(!Docker)"

```

Comment:
the condition should now change to `"(Deploy  != \"docker\")"`

Reply:
Whoops I missed those. Fix now, thanks.